app/src/main/java/com/example/mvvmcodebase/data/ApiService.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.mvvmcodebase.data
+
+import com.example.mvvmcodebase.data.model.CocktailResponse
+import com.example.mvvmcodebase.utils.ApiConstant
+import retrofit2.Call
+import retrofit2.Response
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+interface ApiService {
+
+    @GET(ApiConstant.SEARCH_QUERY)
+    suspend fun getDrinks(@Query("f") letter: String): Response<CocktailResponse>
```

Comment:
Anh tưởng là GsonConverter đã giúp mình convert từ response về object rồi 
```suggestion
    suspend fun getDrinks(@Query("f") letter: String) : CocktailResponse
```

Reply:
Em thấy là dùng response vì mỗi lí do nó ném được message lên viewmodel nếu bị fail thôi ạ ,  em nghĩ e sẽ handle exception ở viewmodel như dưới a nói